.github/workflows/land-blocking.yml

```diff
@@ -296,6 +296,9 @@ jobs:
             }
             // Post test result on original pull request
             try {
+              if (!should_fail) {
+                body += "\n :tada: Land-blocking test passed! :ok_hand:"
```

Comment:
Should we change it to 'Cluster test passed'?

'Land blocking' is a very confusing name - unit tests are also land blocking

Reply:
It's in reference to the name of the GHA workflow, which is "Land-blocking Test". Actually now that you mention it it's a bit confusing... I'll change it to cluster-test